routes/index.js

```diff
@@ -1,7 +1,7 @@
 const express = require( 'express' );
 const router = express.Router();
 const path = require( 'path' )
-const { getAllPosts } = require( '../database' )
+const { db, getAllPosts, getPost, createPost, editPost, deletePost } = require( '../database' )
```

Comment:
I recommend only importing what you need from other modules.  As well, you probably should not need the `db` - you've created wrappers around the direct `db` connection with the creation of the individual `*Post` methods, so should not couple this code to that `db` connection.